### ISSUE TITLE
More specification of attack tree analysis

### ISSUE THREAD

**Issue by jssmith1** [DeveloperLiberationFront/iTrustInterviews#428]
Editor:

>R3 also found the attack tree analysis inadequately specified,

**Comment by jssmith1** [comment]
R3:

>Section 2.4: which does not explain how the authors came up with the strategy tree, nor how the authors came up with failure/success criteria. There is also no mention to a stopping criteria to decide when a strategy was finished. This can also be hardly answered since the data were collected for a different reason.

**Comment by jssmith1** [comment]
Related to #431

**Comment by jssmith1** [comment]
Reading group

### Strategy Analysis
- [x] May be other representations of strategies
- [x] We are introducing this new notation
- [x] How to validate strategy tree #421

**Comment by jssmith1** [comment]
Stopping criteria: leaf nodes were deterministic, that couldn't be broken into meaningful? subparts

**Comment by jssmith1** [comment]
We separated Section 2.4 into two sections and expanded on each of the two new sections, "Strategy Definitions" and "Strategy Extraction." As we mentioned in response to R2, we also added as section describing the additional strategy tree validation we conducted. In the "Strategy Definitions" section we clarify our definitions of strategies and strategy trees. We now describe how strategy trees relate to some other representations of strategies and articulate stopping criteria that were used to determine what to exclude from the trees. Finally, we added a paragraph outlining our success/failure criteria to the "Strategy Extraction" section.
